origami.json

```diff
@@ -2,6 +2,7 @@
 	"description": "FT brand colours: Sass variables and helpers to use and define them",
 	"origamiType": "module",
 	"origamiCategory": "primitives",
+	"keywords": "colours, themes, schemes, pink, salmon, background, palette",
```

Comment:
They'll still match, it's a regex that finds partials, so `col` will match colours/colors and `the` will match `theme/themes` :)